platformtests/aws/default_instance_class_test.go

```diff
@@ -83,6 +83,7 @@ func TestGetDefaultInstanceClass(t *testing.T) {
 		"EU (Stockholm)":             "eu-north-1",
 		"Middle East (Bahrain)":      "me-south-1",
 		"South America (Sao Paulo)":  "sa-east-1",
+		"US West (Los Angeles)":      "us-west-2",
```

Comment:
hmm, not sure why we need this change.

Reply:
https://github.com/openshift/installer/pull/2745/files/b2e979c4dd24106ca1ba8a01e559e8ca85a8e19a..cb2a07377f8dbc1529339d498b80a575649acfc9#diff-925950ad9b0c149dc413dc50e5a4e334R101

Reply:
scrap that, trying to find the CI run, that failed without this

Reply:
we compare these regions against `validation.Regions`; that list must've been updated in a newer version of the aws provider.